src/fsharp/fsi/console.fs

```diff
@@ -17,7 +17,7 @@ module internal ConsoleOptions =
   let fixupRequired = false
 #else
   // Bug 4254 was fixed in Dev11 (Net4.5), so this flag tracks making this fix up version specific.
-  let fixupRequired = not FSharpEnvironment.IsRunningOnNetFx45OrAbove
+  let fixupRequired = not FSharpEnvironment.IsRunningOnNetFx45OrAbove && Type.GetType("Mono.Runtime") = null
```

Comment:
Sure.

I was hoping to get some feedback on the whole thing. Now that we've established it's a bug, is this a good fix? What about .net Core, for example? `IsRunningOnNetFx45OrAbove` is vacuously true both for mono and .Net Core, for example.